### DISCUSSION TITLE
IPBan 1.7.0

### DISCUSSION THREAD

**Discussion by jjxtra** [DigitalRuby/IPBan#167] [General]
- Upgrade to .NET 6
- Big RAM reduction

<hr /><em>This discussion was created from the release <a href='https://github.com/DigitalRuby/IPBan/releases/tag/1.7.0'>IPBan 1.7.0</a>.</em>

**Comment by jjxtra** [comment]
Wanted to get eyes on the .NET 6 release. This should have improved performance and memory usage, but as .NET 6 is still quite new, there is a small amount of chance there could be unforeseen bugs. All the tests are passing which is a good start.

Please chime in with any issues seen in this release. If I don't see anything for a month or so I will promote this to a full release.

**Comment by jjxtra** [comment]
This is now an official release! Very pleased with the binary size, performance and RAM usage, enjoy!

**Reply by s-bourdon** [reply]
Hi!
If I already have IPBan installed and want to update to the latest release, do I simply use the «Easy one click install» procedure?

> [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12; iex ((New-Object System.Net.WebClient).DownloadString('https://raw.githubusercontent.com/DigitalRuby/IPBan/master/IPBanCore/Windows/Scripts/install_latest.ps1'))

Also, do I need to stop any services before attempting an update?

Thank you for your support!

**Reply by jjxtra** [reply]
Just run that from an admin power shell should be good.

**Reply by kimboslice99** [reply]
Just tried this, doesnt work.... complained about the folder existing already, tried to just delete everything so id have a simple one command install, aaaand that was probably a bad idea

`New-Item : An item with the specified name C:\Program Files\IPBan already exists.
At line:86 char:1
+ & mkdir -p $INSTALL_PATH
+ ~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ResourceExists: (C:\Program Files\IPBan:String) [New-Item], IOException
    + FullyQualifiedErrorId : DirectoryExist,Microsoft.PowerShell.Commands.NewItemCommand`

edit: solution was to reboot, delete IPBan folder from \program files\ then run the command
